endpoints/version.go

```diff
@@ -0,0 +1,35 @@
+package endpoints
+
+import (
+	"encoding/json"
+	"net/http"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const versionEndpointValueNotSet = "not-set"
+
+// NewVersionEndpoint returns the latest git tag as the version and commit hash as the revision from which the binary was built
+func NewVersionEndpoint(version, revision string) http.HandlerFunc {
+	if version == "" {
+		version = versionEndpointValueNotSet
+	}
+	if revision == "" {
+		revision = versionEndpointValueNotSet
+	}
+
+	response, err := json.Marshal(struct {
+		Version  string `json:"version"`
+		Revision string `json:"revision"`
+	}{
+		Version:  version,
+		Revision: revision,
```

Comment:
Can you please reverse the order of the Version and Revision to be alphabetical / match PBS?

Reply:
done